### ISSUE TITLE
Migration stalled in queued state

### ISSUE THREAD

**Issue by philthethrill99** [github/gh-gei#585]
Since last night, when running the migrate-repo command, the request migration its stalled in the state queued.

Using gh-gei v0.23

Migrating GHEC > GHEC w/EMU repo migrations

```
gh gei migrate-repo --github-source-org "${GH_SOURCE_ORG}" --github-target-org "${GH_DESTINATION_ORG}" \
    --source-repo "${REPO}"  --target-repo "${REPO}" \
    --github-source-pat "${GH_SOURCE_PAT}" \
    --verbose --wait
```
I attach the log file with GEI_DEBUG_MODE=true
[20220819104423.octoshift.verbose.log](https://github.com/github/gh-gei/files/9383172/20220819104423.octoshift.verbose.log)

I am an Org owner on both the source and target Orgs. Both PATs have all scopes selected.

**Comment by dylan-smith** [comment]
Thanks @philthethrill99. I passed this info onto our team, but the best thing to do is create a github support ticket with this info. A support ticket is the best way to get the right people looking at it, and make sure somebody gets back to you ASAP.

PS - I'm also pleasantly surprised somebody has discovered and is using GEI_DEBUG_MODE so quickly!

**Comment by roferg** [comment]
The team got a fix out on Friday to resolve the stuck migration issue. The migrations processed and we're keeping an eye on the migration attempts occurring this week.

**Comment by dhikrahashim** [comment]
@roferg Could you please let me know the fix. We are getting the same issue when we started migration last night.

`[1:19 AM] [INFO] Migration in progress (ID: DFlLTRmNzQtODdjOS01NmEyOWEwNWEwMTE). State: QUEUED. Waiting 10 seconds...`

**Comment by zikphil** [comment]
Issue still seems to be happening, my migrations have been queued for several hours...

Support ticket: https://support.github.com/ticket/enterprise/10297/2427427

**Comment by timrogers** [comment]
@zikphil Sorry for the delays that you saw last week! The service was exceptionally busy, so we had a long queue of migrations that it took a while to burn down. This certainly isn't normal - our capacity has only been exhausted a couple of times this year - but we are working to increase our migrations capacity for the future.

**Comment by zikphil** [comment]
> @zikphil Sorry for the delays that you saw last week! The service was exceptionally busy, so we had a long queue of migrations that it took a while to burn down. This certainly isn't normal - our capacity has only been exhausted a couple of times this year - but we are working to increase our migrations capacity for the future.

Thank you :)

**Comment by rgavilan** [comment]
We are experiencing the same issue.

We're currently migrating a large volume of repositories (over 200), and many of them are getting stuck in the QUEUED state for extended periods before the migration actually starts. This is significantly slowing down our migration process and making it difficult to plan and execute at scale.

As an example, we launched a batch of 69 repositories at 18:00 CEST, and nearly 3 hours later, only 8 had completed. 

We have opened a support ticket to follow up on this.
